tests/results/a3f571300587d7e8274699f27677b2883522e8d42677c2788c9dcd696a677420_perdu_extra_space/result.json

```diff
@@ -33,7 +33,7 @@
           ],
           [
             "actual_url",
-            "http://perdu.com/"
+            "https://perdu.com/"
```

Comment:
That is not what the .har file contain for that test. I think we should keep the http test, since we have plenty of other https tests.